docs/source/data-publishing/ogcapi-features.rst

```diff
@@ -232,6 +233,11 @@ Must have PostGIS installed.
          table: hotosm_bdi_waterways
          geom_field: foo_geom
 
+This provider has support for the CQL queries as indicated in the Provider table above.
+
+.. seealso::
+  :ref:`cql` for more details on how to use the Common Query Language to filter the collection with specific queries.
```

Comment:
:ref:\`cql\` for more details on how to use the Common Query Language (CQL) to filter the collection with specific queries.

Reply:
This is a copy/paste of the same paragraph in the elasticsearch section. What changes do you want?

Reply:
Add `(CQL)` (the acronym)?